dhclient-ipv6

```diff
@@ -193,9 +193,10 @@ case "$reason" in
     fi
     ;;
   RELEASE)
-    if [ -e /var/run/dhclient6.eth0.pid ]; then
-      echo 'Shutting down dhclient6'
-      kill `cat /var/run/dhclient6.eth0.pid`
+    ext_int_pid="/var/run/dhclient6.${EXT_IFACE}.pid"
```

Comment:
nice catch! Thanks.

However since in this script "int" means "internal" calling this variable `ext_int_pid` is super confusing. Can you rename to `ext_iface_pid`?

Reply:
Ya, no worries. Will do in the morning.